sympy/physics/units/quantities.py

```diff
@@ -88,6 +88,10 @@ def _eval_is_positive(self):
     def _eval_is_constant(self):
         return self.scale_factor.is_constant()
 
+    def _eval_Abs(self):
+        return self.__class__(self.name, self.dimension,
```

Comment:
SymPy's default way of constructing an object is by using the `.func( )` method, in some cases (especially in the logic module), the class constructor may take a hidden parameter that is not shown in the object parameters.

Anyways, now that I look at the code, you should use `Quantity` directly.